tests/test_metadata_utils.py

```diff
@@ -1641,6 +1647,166 @@ def test_add_metadata_and_chunk_examples_with_true_processor(self):
             "url: https://en.wikipedia.org/wiki/Apple ||| An <b>apple [[Malus domestica]]</b> is an edible fruit produced by an <a></a><b class:level1><b class:level2><i class:level3><a></a>apple</i> tree</b></b> (Malus domestica).<|endoftext|>",
         )
 
+    def test_add_metadata_and_chunk_examples_with_true_processor_and_metadata_special_tokens(self):
+        cfg = MetadataConfig()
+        cfg.metadata_list = ["url", "timestamp", "html", "entity"]
+        cfg.max_seq_len = 84
+        cfg.metadata_probability = 1
+        cfg.add_local_metadata_special_tokens_in_prefix = True
+        cfg.metadata_prefix_start_seq = " "
+
+        PROCESSORS["url"] = UrlProcessor
+        PROCESSORS["timestamp"] = TimestampProcessor
+        PROCESSORS["html"] = HtmlProcessor
+        PROCESSORS["entity"] = EntityProcessor
+
+        ds_dict = {
+            key: [self.examples[1][key], self.examples[4][key], self.examples[4][key]]
+            for key in self.examples[0].keys()
+        }
+        ds = Dataset.from_dict(ds_dict)
+
+        mapped_ds = ds.map(
+            functools.partial(add_metadata_and_chunk_examples, tokenizer=self.tokenizer, cfg=cfg),
+            batched=True,
+            remove_columns=ds.column_names,
+            load_from_cache_file=False,
+        )
+
+        self.maxDiff = None
```

Comment:
I'm sorry that I failed to notice this sooner when it was introduced previously. Since it was also [set to None during the `setUp()`](https://github.com/bigscience-workshop/metadata/blob/master/tests/test_metadata_utils.py#L26), would this be a special treatment for certain test functions?

Reply:
Suggestion commited! Thank you :)

Reply:
Honestly, I don't know why I needed this argument is here. If it's there it's probably because I needed it to see all the output when I had an error in the tests, then it's possible that when I added it there was not yet the `self.maxDiff` set to None in the `setUp()` (and that it arrived later in a merge with the `master` branch).